README.md

```diff
@@ -51,16 +57,21 @@ available in the main [server](https://github.com/triton-inference-server/server
 repo. If you don't find your answer there you can ask questions on the
 main Triton [issues page](https://github.com/triton-inference-server/server/issues).
 
-## Building the vLLM Backend
+## Installing the vLLM Backend
 
 There are several ways to install and deploy the vLLM backend.
 
 ### Option 1. Use the Pre-Built Docker Container.
 
-Pull a tritonserver_vllm container with vLLM backend from the
+Pull a `tritonserver:<xx.yy>-vllm-python-py3` container with vLLM backend from the
 [NGC](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver)
-registry. These are available starting in 23.10.
-The tritonserver_vllm container has everything you need to run your vLLM model.
+registry. \<xx.yy\> is the version of Triton that you want to use. Please note,
+that Triton's vLLM container was first published in 23.10 release, so any prior
```

Comment:
I would get rid of the subordinate clause (`, so any prior verson will not work.`). It is redundant and can be unclear for future Triton versions.